pyproject.toml

```diff
@@ -223,6 +225,7 @@ ignore = [
     "TRY003",  # Avoid specifying long messages outside the exception class
     "TRY300",  # Checks for return statements in try blocks
     "TRY301",  # Checks for raise statements within try blocks
+    "NPY201",  # Numpy 2.0 deprecation
```

Comment:
I think we don't need to include `NPY201` here, `pymatgen` now enables all ruff rules by default, and this is the "ignore" section, not the "select" section (where rules are enabled).

Reply:
reverted

Reply:
Thanks! I would probably be able to help looking at the remaining failing tests later once I have access to the Windows PC at home :)